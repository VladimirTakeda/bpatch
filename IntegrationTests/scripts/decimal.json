{
  "dictionary": {
    "decimal": {
      "S": [83],
      "H": [72],
      "text": [116, 101, 120, 116],
      "buy": [98, 117, 121],
      "another": [97, 110, 111, 116, 104, 101, 114],
      "cool": [99, 111, 111, 108]
    }
  },
  "todo": [
    {
      "replace": {
        "S": "H"
      }
    },
    {
      "replace": {
        "text": "buy",
        "another": "cool"
      }
    }
  ]
}