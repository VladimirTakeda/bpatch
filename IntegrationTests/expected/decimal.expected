Home buy to replace.
Here should be cool result
after processing.